src/common/modals/ModOverview.js

```diff
@@ -73,6 +78,35 @@ const ModOverview = ({
     }
   };
 
+  const formatNumber = number => {
+    // Alter numbers larger than 1k
+    if (number >= 1e3) {
```

Comment:
Codacy found an issue: [The numeric literal '1e3' will have at different value at runtime.](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=5870474)